projects/novo-elements/src/utils/form-utils/FormUtils.ts

```diff
@@ -835,24 +835,23 @@ export class FormUtils {
     }
   }
 
-  /**
-   * Get the min start date of a Date base on field data.
-   */
-  private getStartDate(field: any): Date | null {
-    if (field.allowedDateRange) {
-      return this.getStartDateFromRange(field.allowedDateRange);
+  private getEndDateFromRange(dateRange: { maxDate: string; minOffset: number }): Date {
```

Comment:
this should be maxDate and maxOffest, not minOffset.

Reply:
updated